mdx-models/src/main/java/com/mx/path/model/mdx/model/challenges/InfoData.java

```diff
@@ -0,0 +1,15 @@
+package com.mx.path.model.mdx.model.challenges;
+
+import com.mx.path.model.mdx.model.MdxBase;
+
+public final class InfoData extends MdxBase<InfoData> {
```

Comment:
I may have missed the design phase of this change, but to me it seems like the word 'Info' comes from 'Information' which is essentially the same as 'Data', and the word 'InfoData' really seems like 'DataData'.

What exactly is an object named InfoData? Especially since its only field is 'Format', which seems to control font and color.

Reply:
I see in the spec where it says: "info_data	Info	Data fields for INFO type. See [info](https://developer.mx.com/drafts/mdx/challenge/index.html#draft-documentation-info-question) for more information."

but that link doesn't work.

Reply:
@gerfboy technically yes you are correct, I am following the spec here for consistency. Would you like me to update the spec first?

Reply:
Oh, geez, it looks like info_data was added almost 2 years ago. No wonder I don't recall hearing about its introduction.

Reply:
What do you suggest? Should I update the spec or move forward with this name?

Reply:
Looks like this is already a thing in Helios. I'm okay with it.

Reply:
For context:

From @JAHansens 

> Yes I do remember this. the INFO prompt type (not a big fan of the name, but TEXT has a different meaning). Anyway I do remember adding this and the Format options are intentional to allow control over ow the data is presented. Note that the Format uses themed values rather than specific sizes or RGB colors, this keeps the actual display values in control of our designers.